src/Pages/AdminHomePage.js

```diff
@@ -1,43 +1,122 @@
-import React from 'react';
-import {Link} from 'react-router-dom';
+import React,{Component} from 'react';
 import Button from 'react-bootstrap/Button';
-function AdminHomePage() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        
-        <Link to='/CreateSchool'>
-            <Button variant="outline-danger">
-              Create School
-            </Button>
-        </Link>
-          
-        
-        <br />
-        <Link to='/CreateClassroom'>
-            <Button variant="outline-info">
-            Create Classroom
-            </Button>
-        </Link>
+import Table from 'react-bootstrap/Table';
+import InputGroup from 'react-bootstrap/InputGroup';
+import FormControl from 'react-bootstrap/FormControl';
+import {Link} from 'react-router-dom'
 
-        <br />
-
-        <Link to='/AddStudent'>
-            <Button variant="outline-warning">
-            Add Student
-            </Button>
-        </Link>
-
-        
+class SchoolTable extends Component{
```

Comment:
`SchoolTable`  should belong to `SchoolTable` file. Also, rename it to `SchoolTableComponent`